manpages/linux/master/pmreorder/pmreorder.1.md

```diff
@@ -295,6 +295,7 @@ Example of configutation on function level:
     "pmemobj_open"="NoReorderNoCheck",
     "pmemobj_memcpy_persist"="ReorderPartial"
 }
+```
 
 Example of configutation on library level
```

Comment:
mislettering here: should be "configuration" instead of "configutation". I've seen it in other places in this file also.